blocks/library/quote/index.js

```diff
@@ -111,11 +113,24 @@ registerBlockType( 'core/quote', {
 	},
 
 	edit( { attributes, setAttributes, focus, setFocus, mergeBlocks } ) {
-		const { value, citation, style = 1 } = attributes;
+		const { align, value, citation, style = 1 } = attributes;
 		const focusedEditable = focus ? focus.editable || 'value' : null;
 
-		return (
-			<blockquote className={ `blocks-quote blocks-quote-style-${ style }` }>
+		return [
+			focus && (
```

Comment:
I was thinking about the focus check, I guess we could avoid it by hiding the controls using CSS instead of removing them completely from the DOM.